src/js/disability-benefits/components/ClaimDetailLayout.jsx

```diff
@@ -15,13 +15,14 @@ export default class ClaimDetailLayout extends React.Component {
 
     let content;
     if (!loading) {
+      const claimsLink = `your-claims${claim.attributes.open ? '' : '/closed'}`;
```

Comment:
There are other places where you'd want to change this, too, like on the file upload pages. Any claim specific page with breadcrumbs.